app/javascripts/tiles/downloads.jsx

```diff
@@ -59,16 +59,38 @@ var DownloadsTile = React.createClass({
     requests = requests.map(function(months) {
       var start = months[0];
       var end = months[1];
-      return "https://api.npmjs.org/downloads/range/" + start.format("YYYY-MM-DD") + ":" + end.format("YYYY-MM-DD") + "/" + self.props.packageName;
+      const names = self.props.packageName.split(",");
+      return names.map(name => {
+        return "https://api.npmjs.org/downloads/range/" + start.format("YYYY-MM-DD") + ":" + end.format("YYYY-MM-DD") + "/" + name;
+      });
     });
 
-    reduce(requests, [], function(memo, item, callback) {
-      axios.get(item).then(function(response) {
-        var downloads = response.data.downloads;
-        callback(null, memo.concat(downloads));
-      }).catch(callback)
-    }, function(err, downloads) {
+    function getDownloads(memo, items, callback) {
+      const responses = items.map(item => axios.get(item));
+      Promise.all(responses)
+        .then(results => {
+          results.forEach(response => {
+            response.data.downloads.forEach(download => {
+              const existing = memo.find(item => item.day === download.day);
+              if (existing) {
+                existing.downloads += download.downloads;
+              } else {
+                memo.push(download);
+              }
+            });
+          });
+          callback(null, memo);
+        })
+        .catch(error => {
+          console.error(error);
+          callback(error);
+        });
+    }
```

Comment:
this makes it so each "Tile" can specify multiple comma-separated packages. This code downloads the stats for all packages for the Tile and combines them together.